charts/consul/Chart.yaml

```diff
@@ -13,13 +13,13 @@ annotations:
   artifacthub.io/prerelease: false
   artifacthub.io/images: |
     - name: consul
-      image: hashicorp/consul:1.14.5
+      image: hashicorp/consul:1.14.7
     - name: consul-k8s-control-plane
-      image: hashicorp/consul-k8s-control-plane:1.0.6
+      image: hashicorp/consul-k8s-control-plane:1.0.7
     - name: consul-dataplane
-      image: hashicorp/consul-dataplane:1.0.1
+      image: hashicorp/consul-dataplane:1.0.2
     - name: envoy
-      image: envoyproxy/envoy:v1.23.1
+      image: envoyproxy/envoy:v1.24.7
```

Comment:
We have 1.25.x marked as supported in https://developer.hashicorp.com/consul/docs/k8s/compatibility#supported-consul-and-kubernetes-versions; should we be using that instead? (I see @david-yu already approved so this is probably more for my education)

Reply:
Yea we usually bump minor versions of Envoy for each minor version of consul-k8s and consul. This particular value is for the proxy image that is tied to Consul API Gateway, not sure why this wasn't bumped to 1.24.x earlier but that's probably the one we should use. cc @nathancoleman

Reply:
@DanStough It looks like this is still 1.23.1 here https://github.com/hashicorp/consul-k8s/blob/76a5e5b19e0d519a30ff0c50f4863d115e39d19b/charts/consul/values.yaml#L2915. So I'd look to @nathancoleman to see if that needs to be bumped as well. Good catch @zalimeni

Reply:
OK, I'll make the updates. Good catch all!